src/aura/CRLP_FilterGroup/CRLP_FilterGroupHelper.js

```diff
@@ -31,6 +32,18 @@
                 cmp.set("v.isReadOnly", false);
             } else if (mode === "create") {
                 cmp.set("v.isReadOnly", false);
+                cmp.set("v.rollupItems",[]);
+            } else if (mode === "delete") {
+                //verify no rollups use the filter group before deleting
+                var countRollups = cmp.get("v.rollupItems").length;
+
+                if (countRollups == 0) {
+                    this.toggleFilterRuleModal(cmp);
```

Comment:
Looks like you could put `this.toggleFilterRuleModal(cmp);` outside the if block here since it's used on both sides of the `else`